sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+        wizard_id = self.create(vals)
+
+        return {
+            "type": "ir.actions.act_window",
+            "name": "Add Lot to Sale Order",
+            "res_model": self._name,
+            "view_mode": "form",
+            "res_id": wizard_id.id,
+            "target": "new",
+        }
+
+    def action_add_lots_to_sale_order(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `action_add_lots_to_sale_order`

(ANN201)